qa/rpc-tests/excessive.py

```diff
@@ -19,40 +15,79 @@
 import test_framework.script as script
 import pdb
 import sys
-if sys.version_info[0] < 3: raise "Use Python 3"
+if sys.version_info[0] < 3:
+    raise "Use Python 3"
 import logging
 logging.basicConfig(format='%(asctime)s.%(levelname)s: %(message)s', level=logging.INFO)
 
+
+def mostly_sync_mempools(rpc_connections, difference=50, wait=1, verbose=1):
+    """
+    Wait until everybody has the most of the same transactions in their memory
+    pools. There is no guarantee that mempools will ever sync due to the
+    filterInventoryKnown bloom filter.
+    """
+    iterations = 0
+    while True:
+        iterations += 1
+        pool = set(rpc_connections[0].getrawmempool())
+        num_match = 1
+        poolLen = [len(pool)]
+        for i in range(1, len(rpc_connections)):
+            tmp = set(rpc_connections[i].getrawmempool())
+            if tmp == pool:
+                num_match = num_match + 1
+            if iterations > 10 and len(tmp.symmetric_difference(pool)) < difference:
+                num_match = num_match + 1
```

Comment:
so my understanding is that after 10 seconds of sync time you're saying,  "if the mempools haven't sync'd by then then we'll allow a difference which is default of 50" ?  unless there's some other reason,  i'd rather see a default of 0, and then have the script pass in a difference if it's really needed.